be/src/browserToken/middleware.ts

```diff
@@ -71,36 +92,40 @@ const _createBrowserTokenMiddleware = ({
       method: 'POST',
     });
 
-    if (response.status !== 200) {
+    if (fetchResponse.status !== 200) {
       return ctx.throw(500, 'Unexpected status from browser token endpoint');
     }
 
-    const responseBody = (await response.json()) as unknown;
+    const fetchResponseJson = (await fetchResponse.json()) as unknown;
 
-    const responseResult = SeekApiBrowserTokenResponse.validate(responseBody);
+    const result = BrowserTokenResponse.validate(fetchResponseJson);
 
-    if (!responseResult.success) {
+    if (!result.success) {
       return ctx.throw(500, 'Unexpected body from browser token endpoint');
     }
 
-    const expiresInMs = responseResult.value.expires_in * 1000;
+    const response: BrowserTokenResponse = result.value;
+
+    const expiresInMs = response.expires_in * 1000;
+
+    const expiry = new Date(Date.now() + expiresInMs).toISOString();
 
-    const freshToken: BrowserTokenResponse = {
-      authorization: `Bearer ${responseResult.value.access_token}`,
-      expiry: new Date(Date.now() + expiresInMs).toISOString(),
+    const freshItem: CacheItem = {
+      expiry,
+      response,
     };
 
     // Write off the token at its half life. This is a bit mean.
-    tokenCache.set(hirerId, freshToken, expiresInMs / 2);
+    tokenCache.set(cacheKey({ hirerId, scope }), freshItem, expiresInMs / 2);
```

Comment:
Do we still need to do this now that we have the "1 minute discount" added in this PR?

Reply:
That's on the frontend cache; do you think the backend shouldn't cache at all then?

Reply:
No sorry, this is fair :+1: